proposed/http-middleware/middleware.md

```diff
@@ -31,12 +31,11 @@ and the creation of a resulting HTTP Response, as defined by PSR-7.
 
 Middleware using this standard MUST implement the following interface:
 
-- `Psr\Http\ServerMiddleware\MiddlewareInterface`
+- `Psr\Http\Server\ServerMiddlewareInterface`
 
-Middleware dispatching systems using this standard MUST implement the following
-interface:
+Handlers using this standard MUST implement the following interface:
```

Comment:
Should we make it explicit that middlewares should implement `ServerMiddlewareInterface`, even those that do not need the `$handler` (aka `$delegate`) parameter.

The reason for that is that if they don't, interoperability with other middlewares through "dispatchers" (as called in this document, aka pipes/stacks) is lost.